### PR TITLE
Fix SVG attributes in new icons

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Fix SVG attributes to remove "Unkown DOM attributes" warning

Did not find a way to make eslint or typescript check this for us, would be great

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1364** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1364" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1364?utm_source=stack-comment).

File: apps/store/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "../../node_modules/typescript/lib",
```
Comment (robinandeer) [inline apps/store/.vscode/settings.json:2]: Not sure why this file gets added.... 🤔

**Review by robinandeer — APPROVED** [review]
Strange that the vscode settings file get added - not sure we need it

File: apps/store/.vscode/settings.json (hunk 1 above)
Comment (alebedev) [reply apps/store/.vscode/settings.json]: Added it to gitignore to prevent in the future

**Pull request merged**